README.md

```diff
@@ -282,6 +283,15 @@ One thing to note is that OpenAlex will handle data requests faster if the sende
 asreview data snowball input_dataset.csv output_dataset.csv --backward --email my_email@provider.com
 ```
 
+## Sample
+
+This datatool is used to sample n old, random and new records from your dataset by using the `asreview data sample` command. The sampled records are then stored in an output file. This can be useful for detecting concept drift, meaning that the words used for certain concepts change over time. This script assumes that the dataset includes a column named `publication_year`. An example would be:
+
+```bash
+asreview data sample output_dataset.xslx input_dataset.xlsx 50
```

Comment:
I think it makes more sense to have the input dataset before the output dataset. That would also be more in line with the other subcommands.